core-java-modules/core-java-collections-5/src/test/java/com/baeldung/immutables/ImmutableCollectionsExampleTest.java

```diff
@@ -0,0 +1,83 @@
+package com.baeldung.immutables;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ImmutableCollectionsExampleTest {
```

Comment:
Please use assertThrows instead of try catch.

Reply:
Ok